src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -255,6 +267,47 @@ private void readRcaEnabledFromConf() {
         });
   }
 
+  /**
+   * Reads the mutedRCAList value from the rca.conf file, performs validation on the param value
+   * provided and on successful validation, updates the AnalysisGraph with muted RCA value.
+   *
+   * <p>In case all the RCAs in param value are incorrect, return without any update.
+   */
+  private void readAndUpdateMutesRcas() {
+    // If the rca config file has been updated since the lastModifiedTimeInMillisInMemory in memory,
+    // refresh the `muted-rcas` value from rca config file.
+    final RcaConf rcaConf = RcaControllerHelper.pickRcaConfForRole(currentRole);
```

Comment:
The `start()` method already calls `pickRcaConfForRole`. As JSON processing is expensive and this also means that in theory `start()` and `readAndUpdateMutesRcas` can work on different `rca.conf` if it were to be updated between the two calls. 

Long story short, can we store `RcaConf` as a class member after reading it once ?

Reply:
Agreed, makes sense.